site/src/components/WorkspaceStats/createDayString.ts

```diff
@@ -0,0 +1,9 @@
+import dayjs from "dayjs"
+
+/**
+ * Returns a human-readable string describing when the workspace was created
+ * Broken into its own module for testing purposes
+ */
+export function createDayString(createdAt: string): string {
```

Comment:
Should we move this to a `util` and update other instances to use this method? https://github.com/coder/coder/search?q=dayjs%28%29.to

Reply:
gooooood thinking